src/components/Attachments/AttachmentCarousel/extractAttachmentsFromReport.js

```diff
@@ -29,16 +33,39 @@ function extractAttachmentsFromReport(parentReportAction, reportActions) {
                 source: tryResolveUrlFromApiRoot(expensifySource || attribs.src),
                 isAuthTokenRequired: Boolean(expensifySource),
                 file: {name: attribs[CONST.ATTACHMENT_ORIGINAL_FILENAME_ATTRIBUTE]},
+                isReceipt: false,
```

Comment:
I think we want this chunk of code:
```js
            const expensifySource = attribs[CONST.ATTACHMENT_SOURCE_ATTRIBUTE];
            const source = tryResolveUrlFromApiRoot(expensifySource || attribs.src);
            const fileName = attribs[CONST.ATTACHMENT_ORIGINAL_FILENAME_ATTRIBUTE] || FileUtils.getFileName(`${source}`);

            // By iterating actions in chronological order and prepending each attachment
            // we ensure correct order of attachments even across actions with multiple attachments.
            attachments.unshift({
                reportActionID: attribs['data-id'],
                source,
                isAuthTokenRequired: Boolean(expensifySource),
                file: {name: fileName},
                isReceipt: false,
                hasBeenFlagged: attribs['data-flagged'] === 'true',
            });
```

Reply:
Seems like the `FileUtils.getFileName` does not exist now, probably got removed as part of another PR

Reply:
Never mind, that's not on staging.